framework/src/main/java/org/tron/common/application/TronApplicationContext.java

```diff
@@ -23,19 +22,13 @@ public TronApplicationContext(String... basePackages) {
   }
 
   @Override
-  public void destroy() {
-
+  public void doClose() {
+    logger.info("******** start to close ********");
     Application appT = ApplicationFactory.create(this);
     appT.shutdownServices();
     appT.shutdown();
-
-    TronNetService tronNetService = getBean(TronNetService.class);
-    tronNetService.close();
-
-    Manager dbManager = getBean(Manager.class);
-    dbManager.stopRePushThread();
-    dbManager.stopRePushTriggerThread();
-    dbManager.stopFilterProcessThread();
-    super.destroy();
+    super.doClose();
+    logger.info("******** close end ********");
+    FullNode.shutDownSign = true;
```

Comment:
Should the shutDownSign be assigned once the validate executor be closed? Is this reasonable to be here?

Reply:
You can find something in here, https://github.com/tronprotocol/java-tron/pull/4020.